src/components/dialog/faultEvent/FaultEventCreation.schema.tsx

```diff
@@ -36,7 +36,7 @@ export const schema = Yup.object().shape({
     .max(1, "Sequence probability cannot be greater than 1"),
 });
 
-export const rootEventSchema = Yup.object().shape({
+export const eventSchema = Yup.object().shape({
```

Comment:
Do not understand the code, but isn't it useful to know it is related to root event.

Reply:
FaultEventSchema